packages/dds/merge-tree/src/attributionUtils.ts

```diff
@@ -0,0 +1,58 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { assert } from "@fluidframework/core-utils/internal";
+import type { CustomAttributionKey } from "@fluidframework/runtime-definitions/internal";
+
+import type { PropertySet } from "./properties.js";
+
+/**
+ * @internal
+ */
+export const customAttributionKeysPropName: string =
+	"CAK-cf9b6fe4-4c50-4a5d-9045-eb73b886f740";
```

Comment:
I wonder what is "CAK"? Looks like guid with some prefix - why do we need prefix?
Also, I assume we do not care about using actual guids, only (maybe) to isolate user names from "system" names. In some other places we disallow user names to start with "_" (or some other special character) and use that for short system names.

Not sure if it matters, but having long IDs on a lot of small DDSs is not great (tablero default case)